src/ngraph/builder/quantization_util.hpp

```diff
@@ -99,6 +99,59 @@ namespace ngraph
                     (std::pow(2, -24) * static_cast<double>(max_abs32 / max_abs8)));
                 return scale;
             }
+
+            static inline float get_quantize_scale(const std::shared_ptr<Node> min_input,
```

Comment:
We might need a template function here given that `scale` may either be `float` or `double` and the type is tightly controlled by Quantize / Dequantize.  If Quantize input is float, scale is float.  If Quantize input is double, scale is double.  Same goes for Dequantize ouput matching scale.